src/consumed.py

```diff
@@ -210,6 +210,11 @@ def __init__(
         # 930 data
         self.eia930 = BaData(eia930_file)
 
+        # Round 930 data to zero to clear out imputed values.
+        # to mimic behavior of eia930.remove_imputed_ones,
+        # round all values with abs(x) < 1.5 to 0
+        self.eia930.df[self.eia930.df.abs() < 1.5] = 0
```

Comment:
In `eia930.remove_imputed_ones()`, we actually round values to zero decimal places and then replace 1s with zeros, which is almost functionally equivalent to this, except that it wouldn't convert -1 values (which here we are doing by calling `.abs()`. Since in the remove_imputed_ones function we are only dealing with net generation data maybe this shouldn't matter (are there any cases where there might be an imputed -1?) maybe this can be slightly different but just wanted to flag in case we do want this function to be exactly functionally equivalent to the other function.

Reply:
> we actually round values to zero decimal places

I think we should maybe take out the rounding in `remove_imputed_ones` -- seems to me like it throws away info for no reason. For example, even though the raw 930 data doesn't have decimals, if the physics based cleaning were working perfectly, we'd expect it to use info from multiple regions to adjust values and potentially give them meaningful decimal places: for example, if raw export from one region is 10, and import to that region is reported as 11 by its trading partner, we'd want `gridemissions` to set both to 10.5. 

I'm sure it doesn't make a big difference in the results, but it could make a small difference and I don't know what we gain from it.

Reply:
> wouldn't convert -1 values

I think generation is guaranteed to be positive by `gridemissions`, but we could add a check in `remove_imputed_ones` for consistency

Reply:
propose making `remove_imputed_ones` behavior consistent in [dac1e95](https://github.com/singularity-energy/open-grid-emissions/pull/266/commits/dac1e9551338a555e7aae08ef48f6dd32396573a)